<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeckWindow</class>
 <widget class="QDialog" name="DeckWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="Statystyki">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Deck statistics</string>
    </property>
   </widget>
   <widget class="QLabel" name="Regions_T">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>59</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Regions:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Region1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>REGION1</string>
    </property>
   </widget>
   <widget class="QLabel" name="Cost_T">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Average Cost:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Cost">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>AvgCost</string>
    </property>
   </widget>
   <widget class="QLabel" name="Region2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>REGION2</string>
    </property>
   </widget>
   <widget class="QLabel" name="Health_T">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Average Health:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Health">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>AvgHealth</string>
    </property>
   </widget>
   <widget class="QLabel" name="Attack_T">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Average Attack:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Attack">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>AvgAttack</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>230</y>
     <width>271</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="pic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>241</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>230</y>
     <width>241</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="DeleteCard_B">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete one card</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Back_B">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RemoveStack_B">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove stack cards</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ResetDeck_B">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Deck</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>110</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Deck fullfilment</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="deckFill">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>60</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximum">
    <number>40</number>
   </property>
   <property name="value">
    <number>17</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="CardsInDeck">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>230</y>
     <width>341</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
